<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameViewer</class>
 <widget class="QWidget" name="FrameViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Current Frame: 1/1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeFrame">
      <property name="text">
       <string>Remove Frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addFrame">
      <property name="text">
       <string>Add Frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="arrowLeft">
      <property name="text">
       <string>Previous Frame</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="arrowRight">
      <property name="text">
       <string>Next Frame</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
